cdn/dev/js/kmlive.js

```diff
@@ -357,4 +357,4 @@ function toggleDeprecatedVersionDetails() {
   $('#deprecated-old').toggle();
 }
 
-loaded();
+$(loaded);
```

Comment:
Need to wait until html is loaded before running `loaded()`; previously the script was delay-loaded at end of page but now it is loaded in the header.